### DISCUSSION TITLE
how to disconnect CD / CR algorithm

### DISCUSSION THREAD

**Discussion by uriibra** [TUDelft-CNS-ATM/bluesky#471] [Q&A] [answered]
Hi, i have to do some simulations on diferent trajectories, some of them have conflicts, and their trajecotries get modified by the conflict resolution system. I was wondering how i can disconnect this system, is there any way to do it?

Thank you in advance

**Comment by ProfHoekstra** [comment, marked as answer]
Yes, ASAS OFF should do that. Or RESO OFF if you want the conflict detection but not the conflict resolution

Best regards/Met vriendelijke groeten,
    Jacco Hoekstra


On 7 Jul 2023, at 17:10, uriibra ***@***.***> wrote:

﻿

Hi, i have to do some simulations on diferent trajectories, some of them have conflicts, and their trajecotries get modified by the conflict resolution system. I was wondering how i can disconnect this system, is there any way to do it?

Thank you in advance

**Reply by jooste** [reply]
Note that conflict resolution is off by default. It could therefore be that you are seeing something else.

**Comment by ProfHoekstra** [comment]
…like aircraft performance limitations. They can also cause the aircraft to do something else than the flight plan, e.g. adjust the climb rate or airspeed.

Best regards,
    Jacco Hoekstra


On 8 Jul 2023, at 19:25, Joost Ellerbroek ***@***.***> wrote:

﻿

Note that conflict resolution is off by default. It could therefore be that you are seeing something else.

**Comment by uriibra** [comment]
Thank you for your Suggestions, i will try everything!
